src/registries/npm-registry.js

```diff
@@ -12,7 +12,7 @@ import {addSuffix, removePrefix} from '../util/misc';
 import isRequestToRegistry from './is-request-to-registry.js';
 
 const defaults = require('defaults');
-const userHome = require('user-home');
+const userHome = require('os').homedir();
```

Comment:
Would you like me to use that internal module for all uses of `os.homedir`?  What would you like me to name that module?